service/storage/operations_test.go

```diff
@@ -0,0 +1,321 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package storage
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+
+	"github.com/optakt/flow-dps/testing/helpers"
+)
+
+func TestSaveAndRetrieve_First(t *testing.T) {
+	db := helpers.InMemoryDB(t)
+	defer db.Close()
+
+	t.Run("save first height", func(t *testing.T) {
+		txn := db.NewTransaction(true)
+		err := SaveFirst(42)(txn)
+
+		assert.NoError(t, err)
+
+		err = txn.Commit()
+		require.NoError(t, err)
```

Comment:
Overall, I'm not too sure why you explicitly create the transaction each time? Why not just use `err := db.Update(SaveFirst(42))`?